ydb/tests/tools/fqrun/configuration/fq_config.conf

```diff
@@ -69,6 +73,157 @@ DbPool {
   }
 }
 
+Gateways {
+  Enabled: true
+
+  Dq {
+    DefaultSettings {
+      Name: "EnableComputeActor"
+      Value: "1"
+    }
+    DefaultSettings {
+      Name: "ComputeActorType"
+      Value: "async"
+    }
+    DefaultSettings {
+      Name: "AnalyzeQuery"
+      Value: "true"
+    }
+    DefaultSettings {
+      Name: "MaxTasksPerStage"
+      Value: "200"
+    }
+    DefaultSettings {
+      Name: "MaxTasksPerOperation"
+      Value: "200"
+    }
+    DefaultSettings {
+      Name: "EnableInsert"
+      Value: "true"
+    }
+    DefaultSettings {
+      Name: "_EnablePrecompute"
+      Value: "true"
+    }
+    DefaultSettings {
+      Name: "UseAggPhases"
+      Value: "true"
+    }
+    DefaultSettings {
+      Name: "HashJoinMode"
+      Value: "grace"
+    }
+    DefaultSettings {
+      Name: "UseFastPickleTransport"
+      Value: "true"
+    }
+    DefaultSettings {
+      Name: "UseOOBTransport"
+      Value: "true"
+    }
+    DefaultSettings {
+      Name: "UseWideChannels"
+      Value: "true"
```

Comment:
Not *completely* sure, but I think this is not enabled in real installation (and sample flags should match them)